point

```diff
@@ -86,7 +86,7 @@ if (program.verbose) ctx.config.log.level = 'trace';
 
 // ------------------- Init Logger ----------------- //
 
-ctx.log = logger.child({account: ctx.config.client.wallet.account});
+ctx.log = logger.child({module: "main"}); // TODO: account
```

Comment:
1. Why didn't `eslint` show the double quotes error here?
2. Let's rename this module to `point` instead of `main`